feature/gribi/ate_tests/ipv4_entry_test/ipv4_entry_test.go

```diff
@@ -304,8 +304,8 @@ func TestIPv4Entry(t *testing.T) {
 						d := gnmi.OC()
 						p2 := dut.Port(t, "port2")
 						p3 := dut.Port(t, "port3")
-						gnmi.Update(t, dut, d.Interface(p2.Name()).Config(), dutPort2DummyIP.NewOCInterface(p2.Name()))
-						gnmi.Update(t, dut, d.Interface(p3.Name()).Config(), dutPort3DummyIP.NewOCInterface(p3.Name()))
+						gnmi.Replace(t, dut, d.Interface(p2.Name()).Config(), dutPort2DummyIP.NewOCInterface(p2.Name()))
+						gnmi.Replace(t, dut, d.Interface(p3.Name()).Config(), dutPort3DummyIP.NewOCInterface(p3.Name()))
```

Comment:
Hi @prakashbadri-arista This is not correct. With this change, for Arista we are basically updating the primary IP of the Interface itself. 

This would make the test wrong.

The idea behind using gNMI.Update() here is so that a `secondary` ip address is added to the interface, which can then be used in gRIBI next hop.

Instead we should be adding a Magic IP and interfaces in 
`/network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/interface-ref`

Reply:
Hi Ankur,
I have updated the review accordingly. Please take a look.